TheJumpCloud/support
# Sa 2725 commands gallery pipeline

## Issues
* [SA-2725](https://jumpcloud.atlassian.net/browse/SA-2725) - Commands Gallery Pipeline

## What does this solve?

Commands Gallery files are now validated to a degree. A `commands.json` file is generated when new files or files are modified in the command gallery. Null values or values which are not compiled into the `commands.json` file throw an error in our CI pipeline. 

## Is there anything particularly tricky?

## How should this be tested?

Within this repository commit a new command, the commands.json file should be re-generated with the contents of the file. 
Pull the commits and verify the commands.json file contains the information from your command.
Delete your command from the gallery and commit the change to this branch. the `commands.json` file should be updated and remove file on the next automated commit.

## Screenshots
